torch/onnx/symbolic_opset9.py

```diff
@@ -931,6 +931,15 @@ def selu(g, input):
 
 @parse_args('v', 'i', 'v')
 def index_select(g, self, dim, index):
+    index_const = sym_help._maybe_get_scalar(index)
+    if not sym_help._is_value(index_const):
+        index = g.op("Constant", value_t=torch.LongTensor([index_const]))
+    elif index.type().kind() == "CompleteTensorType" or index.type().kind() == "DimensionedTensorType":
```

Comment:
This pattern is pretty common, can we extract it as a helper function?

Reply:
helper function has been added and used in a few places.